### DISCUSSION TITLE
Drivers position (not GPS but on grid) per lap

### DISCUSSION THREAD

**Discussion by walteij** [theOehrly/Fast-F1#198] [Q&A] [answered]
Is there an easy way to see what the position on the grid of a driver is per lap?
I know that we have the gridposition (so startinggrid)  and position at the end of the race, but it would be great to see the positions of the drivers per lap.

The reason for this, is because I want to be able to create something similar to this: https://twitter.com/f1visualized/status/1518324677218869248

**Comment by siddharth-tewari** [comment]
@walteij You can probably check out <a href=http://ergast.com/api/f1/2022/9/laps?>http://ergast.com/api/f1/2022/9/laps</a>

**Comment by theOehrly** [comment, marked as answer]
This is currently not implemented. I was a bit surprised by this myself just now 😅 And no one has ever asked for this before too. I guess I'll add it to the todo list then. This really should be added.

**Reply by siddharth-tewari** [reply]
I did asked for something like this here.
https://github.com/theOehrly/Fast-F1/discussions/96#discussion-3807030

**Reply by f1multiviewer** [reply]
You might be able to use `LapSeries.json(Stream)` for this: https://livetiming.formula1.com/static/2022/2022-07-31_Hungarian_Grand_Prix/2022-07-31_Race/LapSeries.json
